templates/monitoring/fuseonline/addon-ops-api-dashboard.yml

```diff
@@ -0,0 +1,3516 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: syndesis-infra-api-dashboard
+  namespace: {{ index .Params "Namespace" }}
+  labels:
+    app: syndesis
```

Comment:
Do these new monitoring resources also need the `monitoring-key=middleware` label or is that just the namespace for them to be picked up? I see some are missing it.

Reply:
@aidenkeating I see all dashboards and alerts are present based on your screenshot. So is the `monitoring-key=middleware` no longer needed?